src/store/highlight/__tests__/reducer-test.ts

```diff
@@ -27,6 +27,28 @@ describe('store/highlight/reducer', () => {
 
             expect(newState.selection).toEqual({ ...state.selection, location: 2 });
         });
+
+        test('should set selection in IE/Edge', () => {
```

Comment:
This seems like a test more suited for the highlight store actions than the reducer?